src/Components/CardList.tsx

```diff
@@ -1,28 +1,123 @@
-import { useContext, useEffect } from 'react'
+import React, { useContext, useEffect, useState } from 'react'
+import styled from 'styled-components'
 
 import { Card } from './Card'
 import { EmptyList } from './EmptyList'
 import { ServicePreview } from '../../pages/index'
 import { FilterContext } from '../context/FilterContext'
 import { SaveContext } from '../context/SaveContext'
+import { useRemember } from '../hooks/remember'
+import { useScrollRemember } from '../hooks/scrollRemember'
+import { ButtonBase } from './ButtonBase'
+
+const ITEMS_PER_PAGE = 20
 
 interface CardListProps {
   services: Array<ServicePreview>
   listType: 'filtered' | 'saved'
-  onLoad?(): void
+}
+
+const NavContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  margin: 1rem 0;
+  align-items: center;
+`
+
+const NavigationButton = styled(ButtonBase)`
+  display: inline-block;
+
+  &:disabled {
+    opacity: 0;
```

Comment:
I can still tell this is here by hover 🕵️ 
I'm aiming to merge this with moving he filter here - so fine to leave for now.

Reply:
Eh, that's probably fine. We're expecting mostly mobile users (who can't hover), and the 'disabled' state should be interpreted by screenreaders to ignore it.